### ISSUE TITLE
Owner inactive

### ISSUE THREAD

**Issue by CIenthusiast** [Unitech/pm2#5736]
The last few months i have seen more and more Issues that are implying that Unitech / Keymetrics are not available anymore.

As of today no requests to all of the available email-adresses aswell as the request-form has been answerd and i can see other users share the same problem in 5726, 5702, 5720 or 5708.

I would like to have this centralized issue to keep track of any news and on how the community will deal with this. For now theres no technical problem (as long as you use the free-version) since the code remains maintained.

**Comment by CIenthusiast** [comment]
#5726, #5702, #5720 #5708

**Comment by CIenthusiast** [comment]
@Unitech

**Comment by lvilasboas-dti** [comment]
I agree and share the sentiment. Would love to hear if anyone has any strong alternative suggestions that could replace pm2.

**Comment by zhiyuanzhai** [comment]
Does anyone know about Keymetrics? I hardly find any open information about this company...

**Comment by CIenthusiast** [comment]
![image](https://github.com/Unitech/pm2/assets/131581411/23f9750b-3e80-4253-b91c-24e5c286d7de)
He seems to be active tho.

**Comment by jlippold** [comment]
https://github.com/Unitech/pm2/issues/5792

**Comment by jlippold** [comment]
So i guess using pm2 paid with keymetrics isnt a thing? I can't sign up at all, and after seeing this thread, I feel like I need a new solution. Has anyone switched to a new platform?

**Comment by bguerout** [comment]
I subscribed to Keymetrics for one month in September 2024 and then canceled.

I did not receive any invoice, and at the end of November, two charges were made to my card.

The bank informed me that the payment token linked to PM2 PLUS appears to have been used to pay for a limousine in the Netherlands.

I contacted Keymetrics, but they never responded.

**Comment by Unitech** [comment]
C'est quoi ce délire ?

**Comment by bguerout** [comment]
Je ne sais pas, j'avais utilisé le service en 2023 et ca n'avait pas posé de problème.

J'ai contacté Keymetrics (sales@keymetrics.io, support@keymetrics.io, security@keymetrics.io) et je n'ai eu aucune réponse.

**Comment by bguerout** [comment]
Je ne sais pas ce qu'il se passe, vous avez eu l'information, c'est le principal.
Je supprime le message précédent.

**Comment by bguerout** [comment]
Ok apologies for the previous message, this was a mistake, don't understand why the bank tells that me that but the charge was made from another keymetrics account
